packages/insomnia-app/app/models/helpers/settings.ts

```diff
@@ -39,6 +38,10 @@ export const getLocalDevConfigFilePath = () => (
   isDevelopment() ? '../../packages/insomnia-app/app' as string : undefined
```

Comment:
Not related to this PR but should we use the path util instead:
```suggestion
  isDevelopment() ? path.join(__dirname, '..', '..', 'packages', 'insomnia-app', 'app') as string : undefined
```

Reply:
I think it's related _enough_ since this literal path appears on the screen to users as part of the change to the modal made in this PR.  it's not going to ever fail since it goes through `resolve`, but yes, you're right, this should be fixed.

Reply:
update, heh.  it didn't work.. not sure why.  but yes, I agree, we can handle later